.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Task
+about: Template for a new task
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### Task
+<!-- Describe what needs to be done -->
```

Comment:
this syntax means the line won't show on the issue if you don't bother to delete it

Reply:
oh nice!